tests/unittest/_torch/speculative/test_eagle3.py

```
@@ -51,11 +51,11 @@ def enforce_single_worker(monkeypatch):
     ])
 @pytest.mark.high_cuda_memory
-def test_llama_eagle3(use_cuda_graph: bool, attn_backend: str,
-                      disable_overlap_scheduler: bool, enable_block_reuse: bool,
-                      use_one_model: bool, enable_chunked_prefill: bool,
-                      use_chain_drafter: bool, multi_batch: bool,
-                      attention_dp: bool, request):
+def test_foo(use_cuda_graph: bool, attn_backend: str,
+             disable_overlap_scheduler: bool, enable_block_reuse: bool,
+             use_one_model: bool, enable_chunked_prefill: bool,
+             use_chain_drafter: bool, multi_batch: bool, attention_dp: bool,
+             request):
```

Ruff (0.14.2)

Unused function argument: `request`

(ARG001)